hwdb.d/60-keyboard.hwdb

```diff
@@ -523,6 +523,10 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnGeminiDevices:pnNC14V1006:*
 # Genius
 ###########################################################
 
+# HaierComputer 7G-Series
```

Comment:
Hm, I am not familiar with the company, but please make a `Haier` section and put it there:
```
###########################################################
# Haier
###########################################################

# 7G-Series
evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHaierComputer:pn7G-Series:*
 KEYBOARD_KEY_91=wlan
```